iot-e2e-tests/android/runInstrumentationTests.py

```diff
@@ -0,0 +1,21 @@
+import subprocess
+import os
+import time
+import sys
+
+env=dict(os.environ, PATH="path")
+#cmd = "adb -s "+sys.argv[1]+" shell am instrument -w -r -e debug false -e IOTHUB_CONNECTION_STRING '"+env["IOTHUB_CONNECTION_STRING"]+"' -e IOTHUB_E2E_X509_CERT_BASE64 "+env["IOTHUB_E2E_X509_CERT_BASE64"]+" -e IOTHUB_E2E_X509_PK_BASE64 "+env["IOTHUB_E2E_X509_PRIVATE_KEY_BASE64"]+" -e IOTHUB_E2E_X509_THUMBPRINT "+env["IOTHUB_E2E_X509_THUMBPRINT"]+"  -e package com.microsoft.azure.sdk.iot.android com.iothub.azure.microsoft.com.androidsample.test/android.support.test.runner.AndroidJUnitRunner"
+cmd = "adb -s "+sys.argv[1]+" shell am instrument -w -r -e debug false -e package 'com.microsoft.azure.sdk.iot.androidthings' com.microsoft.azure.sdk.iot.androidthings.test/android.support.test.runner.AndroidJUnitRunner"
+print(cmd)
+p = subprocess.Popen(cmd,  stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf8')
+(out, err) = p.communicate()
+print(out)
+print(err)
+os.popen("taskkill /IM cmd.exe /FI \"WINDOWTITLE eq C:\windows\system32\cmd.exe - emulator*\"")
```

Comment:
Make sure your print stmnts are more descriptive to understand during debugging

Reply:
currently, we are capturing whatever is printed on the testlog which is the same as what we have when we are testing with AppCenter.